src/js/modules/drawOdds.js

```diff
@@ -0,0 +1,48 @@
+var math = require('mathjs');
+var myData = { };
+
+/***
+ * Given a deck state, calculate a range of odds
+ * @param cardsInDeck How many cards remain in the deck
+ * @param targetCardsInDeck How many cards in the deck represent a success
+ * @param cardsToDraw How many cards to draw
+ * @returns {Array} An array of probabilities, where the nth index represents the probability of exactly n successes
+ */
+var oddsOfSuccess = function(cardsInDeck, targetCardsInDeck, cardsToDraw) {
```

Comment:
Oh, so this function returns an array of successes? That's cool. I still need to figure out how to use this then :open_mouth: